xAPI.md

```diff
@@ -306,10 +306,21 @@ be unintuitive and/or lengthy to dissect into a list of requirements.
 
 ## 4.0 Binding to JavaScript Object Notation (JSON)
 
+This specification makes a specific binding to JSON for most of the data structures allowed.  There are some 
```

Comment:
this sentence needs tightening. It uses strange wordings such as saying JSON "is the current structure" (JSON isn't a structure) and is "the recommended use for document types" (JSON isn't a 'use')